prime-router/settings/organizations-prod.yml

```
@@ -309,7 +328,61 @@
         type: SFTP_LEGACY
         host: sftp-edts.hhs.texas.gov
         port: 22
+        filePath: ./USDS_IN
+
+- name: gu-doh
+  description: Guam Department of Health
+  jurisdiction: STATE
+  stateCode: GU
+  receivers:
+    - name: elr
+      organizationName: gu-doh
+      topic: covid-19
+      jurisdictionalFilter:
+        - matches(ordering_facility_state, GU)
+      translation:
+        type: HL7
+        useBatchHeaders: true
+        receivingApplicationName: GUDOH
+        receivingFacilityName: GUDOH
+      transport:
+        type: SFTP
+        host: sftp-west.inductivehealth.com
+        port: 22
         filePath: ./
+      timing:
+        operation: MERGE
+        numberPerDay: 12 # every two hours
+        initialTime: 00:00
```

```suggestion
        initialTime: 01:15
```
To match timing of other every-two-hour deliveries, again more likely to leave us free of data-in-transit